app/mesh/changelog.md

```diff
@@ -5,6 +5,17 @@ layout: reference
 description: "{{site.mesh_product_name}} release notes"
 products:
     - mesh
+breadcrumbs:
+    - /mesh/
+
+tags:
+    - changelog
+    - release-notes
```

Comment:
We should pick one tag, changelog or release-notes, for consistency. I don't really have a preference here.